agents/gemini_data_analytics/intro_gemini_data_analytics_http.ipynb

```
@@ -0,0 +1,1240 @@
+        "            {\n",
+        "                \"projectId\": \"bigquery-public-data\",\n",
+        "                \"datasetId\": \"san_francisco\",\n",
+        "                \"tableId\": \"street_trees\",\n",
+        "            },\n",
+        "            # Add more table references here\n",
+        "        ]\n",
+        "    }\n",
+        "}\n",
+        "\n",
+        "# optional example queries (only leveraged for BigQuery datasources currently)\n",
+        "example_queries = [\n",
```

Ruff (0.13.3)

Found commented-out code

Remove commented-out code

(ERA001)